recipes/muparser/all/conanfile.py

```diff
@@ -41,21 +41,16 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def validate(self):
-        if self.info.settings.compiler.get_safe("cppstd"):
+        if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, 11)
-        if self.info.options.with_openmp:
-            self.output.warn("Conan package for OpenMP is not available, this package will be used from system.")
```

Comment:
I think the warning is good too keep just as a disclaimer

Reply:
This warning makes little sense. OpenMP comes from your compiler toolchain.